use t::Util;
use_ok "Rno::Exception";

done_testing;
